src/reports/test.js

```diff
@@ -75,4 +75,14 @@ export default class TestReport {
 	getAssertions() {
 		return this.assertions.slice();
 	}
+
+	// Remove actual and expected values from assertions. This is to prevent
+	// leaking memory throughout a test suite.
+	slimAssertions() {
+		this.assertions = this.assertions.map( assertion => {
```

Comment:
I think using `this.assertions.forEach` makes more sense, since we're not mutating either the referenced array of assertions nor any object references within the array.

I'd love to see a test as well, if at all possible (maybe check the assertion reports at end of module?).